tests/validation/tests/v3_api/resource/terraform/rke2/master/optional_write_files.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# This script pulls raw files and writes to specified locations.
+# For example, it can be used to write HelmChartConfig or custom PSA files.
```

Comment:
I meant this script.

Reply:
Should be fixed now! Thanks for the suggestion